### ISSUE TITLE
Time slider doesn't show value of numeric columns only date types

### ISSUE THREAD

**Issue by andrewxhill** [CartoDB/torque#200]
Take a look,

https://team.cartodb.com/u/andrew/tables/andrew.all_month_delete/public/map

The column used as my date column is a numeric one ranging from 2000-2010. You can see it in the table view as 'yoy'. 

The expected behavior would be that the slider column would show 2000-2010, instead it shows the animation step which doesn't make much sense.

**Comment by fdansv** [comment]
In order to populate the timeslider with any values other than the corresponding step number, the layergroup needs to return the start / end values, so for this to happen we'd have to modify it from Windshaft, right @rochoa ?

**Comment by rochoa** [comment]
What do you mean by start/end values?

The tiler already returns start and end time, check this example: https://rochoa.cartodb.com/api/v1/map/named/tpl_9d99e242_5f9a_11e4_bc5f_0e853d047bba/jsonp?callback=cb

**Comment by fdansv** [comment]
Yeah but I mean, for non-date fields. 2000-2010 are numerical values, for which we don't provide start/end in the layergroup right?

**Comment by javisantana** [comment]
It never worked like that but since we have the start end values we could interpolate in the same way we do with timestamps.

we need to be careful with this change since it changes the way time slider works for already working maps

**Comment by fdansv** [comment]
> since we have the start end values

I seee, apologies, I was under the impression that we only returned start/end for datetypes. This makes it much more visible. Cool!
